tox.ini

```diff
@@ -30,3 +32,19 @@ commands=
     env
     pip install . --force --upgrade
     python -m pytest {posargs: -lvvvs tests} --workers auto
+
+[testenv:xdist]
+setenv =
+    USE_TEST_UTILS2 = true
+;    mac
```

Comment:
Would you please elaborate on this comment e.g., `MacOS requires the following environment variable`